e2e/cypress/integration/ui/compliance/01_ssh_scan_job.spec.ts

```diff
@@ -2,7 +2,7 @@ import { Base64 } from 'js-base64';
 
 describe('create a manual node ssh scan job and cleanup after', () => {
   before(() => {
-    cy.adminLogin('/');
+    cy.adminLogin('/settings/node-credentials');
```

Comment:
Took off some extra page loading time by directly navigating to the first tested page rather than opening the event feed, then the settings page, then node credentials